### DISCUSSION TITLE
Not able to Create Case | Service Cloud Connector

### DISCUSSION THREAD

**Discussion by PremSolanki** [SalesforceCommerceCloud/sfcc-ci#409] [Team Posts]
Hi All

I'm using the sfcc-connector : release branch which shows that it has the functionality of creation of case at service cloud.
I followed all the steps and able to push the data from commerce cloud through OCAPI 
Data format : 
`{"first_name":"PremPre","last_name":"Solanki","email":"test098@yopmail.com","phone":"12345678","case_type":"ContactUs","order_no":"000100","case_sub_type":"My Account","subject":"My Account","description":"ddqd","contact_id":"001f400000vORBHAA4","customer_id":"bcJE3pt3a9w3TltB5f66l5iH68"}
`
but we are not able to see the data at service cloud.Is there any configuration which needs to handle at service cloud side.

Thanks

**Comment by pravngaur** [comment]
Hi,

As you mentioned, are you sending this request to ocapi?
